### PR TITLE
Remove gce_* prefix

### PR DESCRIPTION
**Pull request by electrocucaracha**
The `gce_` prefix used for some bash scripts causes misunderstanding of its use. This change renames the files and their usage. It's necessary to fix the terraform script to make sure the operation is not affected.

Resolves: https://github.com/nephio-project/nephio/issues/298

\cc @johnbelamaric @gvbalaji

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -29,8 +29,8 @@ presubmits:
           args:
             - "-c"
             - |
-              cd e2e/terraform && terraform init && terraform plan && \
-              terraform apply -auto-approve; terraform destroy -auto-approve
+              set -eE; cd e2e/terraform; trap 'terraform destroy -auto-approve' EXIT;
```
Comment (vjayaramrh) [inline .prow.yaml:32]: there is no gce_* renaming here on L32 and L33

File: .prow.yaml [hunk 2]
```diff
@@ -225,8 +225,8 @@ periodics:
...
```
Comment (vjayaramrh) [inline .prow.yaml:228]: there is no gce_* renaming here on L228 and L229

**Comment by liamfallon** [comment]
/approve

These occurrences match what I found as well.

**Review by gvbalaji — APPROVED** [review]
/lgtm
/approve

**Pull request merged**